### PR TITLE
changed openai embedder from chromadb to langchain #1455

### PR DESCRIPTION
**Pull request by SumantxD**
changed openai embedder from chromadb to langchain #1455

### CODE REVIEW

File: embedchain/embedder/openai.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import os
 from typing import Optional
 
-from chromadb.utils.embedding_functions import OpenAIEmbeddingFunction
+from langchain_community.embeddings import OpenAIEmbeddings
```
Comment (deshraj) [inline embedchain/embedder/openai.py:4]: Instead of using from langchain_community, can we directly use the OpenAI APIs here for generating embeddings?

Comment (sumant-spirinova) [reply embedchain/embedder/openai.py:4]: sure,thx for the suggestion.

Comment (SumantxD) [reply embedchain/embedder/openai.py]: @deshraj thx for the suggestion, fixed it!!

**Comment by Dev-Khant** [comment]
@SumantxD Can you please resolve the conflicts?

**Comment by Dev-Khant** [comment]
@SumantxD Please resolve the conflict here so that we can run the tests.

**Comment by Dev-Khant** [comment]
@SumantxD Are you working on this?

**Comment by Dev-Khant** [comment]
Closing this PR as it has been Stale. Please feel free to open a new one if required.

**Pull request closed without merging**